README.md

```diff
@@ -171,9 +171,9 @@ Fides (*fee-dhez*, Latin: Fidēs) is an open-source tool that allows you to easi
 
     <details>
 
-      <summary>Run <code>fidesctl evaluate</code> again</summary>
+      <summary>Run <code>fidesctl evaluate demo_resources/</code> </summary>
 
-        Re-run `fidesctl evaluate demo_resources` which will cause an evaluation failure! This is because your privacy policy has 1 rule that should fail if any system uses contact information for marketing purposes, and you've just updated your marketing system to start using contact information for marketing purposes.
+      Re-run `fidesctl evaluate demo_resources` which will cause an evaluation failure! This is because your privacy policy has 1 rule that should fail if any system uses contact information for marketing purposes, and you've just updated your marketing system to start using contact information for marketing purposes.
```

Comment:
I might simplify/clean up this second sentence a little for clarity - 

Running `fidesctl evaluate demo_resources` now causes an evaluation failure. Your privacy policy's 'Reject Direct Marketing' rule disallows collecting contact information for marketing purposes, and flagged adding this data category and use to your `privacy_declarations`.